include/mygettext/gettext.h

```diff
@@ -20,10 +20,11 @@
 #pragma once
 
 #include "LocaleInfo.h"
-#include <iconv.h>
 #include <map>
 #include <string>
 
+namespace s25gettext {
```

Comment:
the namesace should be `mygettext`, or does this conflict with the function?

Reply:
Yes it does conflict. How about putting everything in a namespace and adjusting the defines to use `::mygettext::gettext` (renaming `mygettext` function to `gettext`)?
The class should also be renamed to be more specific. But couldn't come up with anything better then `Registry` or `State` or so. It combines at least 3 functionalities which isn't great for naming (and in general)

Reply:
sounds good

Reply:
https://github.com/Return-To-The-Roots/mygettext/pull/8/commits/3638ea54f10d8c3a45a8e1bcc2f4af31e96f865b

Reply:
https://github.com/Return-To-The-Roots/mygettext/pull/8/commits/3638ea54f10d8c3a45a8e1bcc2f4af31e96f865b